thoth/user_api/api_v1.py

```diff
@@ -246,12 +247,12 @@ def get_provenance_python_status(analysis_id: str):
 
 def post_advise_python(
     input: dict,
-    recommendation_type: str,
-    count: int = None,
-    limit: int = None,
-    limit_latest_versions: int = None,
-    origin: str = None,
-    is_s2i: bool = None,
+    recommendation_type: typing.Optional[str] = None,
+    count: typing.Optional[int] = None,
+    limit: typing.Optional[int] = None,
+    limit_latest_versions: typing.Optional[int] = None,
+    origin: typing.Optional[str] = None,
+    source_type: typing.Optional[ThothAdviserIntegrationEnum.value] = None,
```

Comment:
Isn't `.value` an instance variable which returns the string representation of the enum?

Reply:
updated :)